sdc/tests/test_io.py

```diff
@@ -663,6 +663,8 @@ def test_impl():
 
         return test_impl
 
+    @skip_numba_jit("Lowering error while returning df from objmode:\n"
```

Comment:
Are we going to do something with this?

Reply:
@Hardcode84 Yes, the fix is rather simple - following lines are absent in sdc/hiframes/boxing.py:
```
                     elif ty_series == string_array_type:
                         native_val = unbox_str_series(string_array_type, series_obj, c)
+                    elif isinstance(ty_series, Categorical):
+                        native_val = unbox_Categorical(ty_series, arr_obj, c)
```
but I want to make it in a separate PR (add missed tests and move DF column unboxing into separate function). I'll make a new PR shortly.

Reply:
ok